tools/RcclReplayer/rcclReplayer.cpp

```diff
@@ -180,31 +180,39 @@ void dataToCsv(GroupCall const& gc, std::ofstream &datafile, double runTime)
 
 void ParseCollectives(char const* logFilename, bool isFirstRank, CollectiveCalls& cc)
 {
-  bool verbose = isFirstRank && (getenv("VERBOSE") != NULL);
+  //bool verbose = isFirstRank && (getenv("VERBOSE") != NULL);
   cc.globalRankComms.clear();
   cc.globalRankComms.resize(cc.numGlobalRanks);
   cc.groupCalls.clear();
 
-  FILE* fp = fopen(logFilename, "r");
+  std::ifstream fp(logFilename, std::ios::binary);
   if (!fp) {
     printf("[ERROR] Unable to open file %s\n", logFilename);
     exit(-1);
   }
 
-  char line[2048];
-  LineItem li;
+  const size_t size = sizeof(LineItem) + 1;
+  char line[size]; //size of collectivecall struct
```

Comment:
This may generate a variable-length-array warning (not sure). To avoid, I think it should be:
```suggestion
  constexpr size_t size = sizeof(LineItem) + 1;
  char line[size]; //size of collectivecall struct
```